.circleci/config.yml

```diff
@@ -142,51 +141,49 @@ jobs:
           command: brew install swiftlint
       - install-dependencies
       - run:
-          name: Build AppleTV, WatchOS, and macOS
+          name: Build tvOS and watchOS
           command: bundle exec fastlane build_tv_watch_mac
 
   swift_apitest:
     macos:
-      xcode: "12.5.1"
+      xcode: "13.1.0"
     working_directory: ~/purchases-ios
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
       - update-spm-integration-commit
       - run:
           name: Build SwiftAPITester
-          command: xcodebuild -workspace APITesters/APITesters.xcworkspace -scheme SwiftAPITester
+          command: |
+              bundle exec fastlane build_swift_api_tester
```

Comment:
it's so that we can write the command in the next line, for _slightly_ more readability. Also, if you have more than one command, you can just write the next one in the next line. I think that's why I did it this way, I used to have another line and then removed it

Reply:
not exactly a huge gain and it's up to personal preference, happy to go back to same line